cmd/podman/trust.go

```diff
@@ -269,3 +267,66 @@ func isValidTrustType(t string) bool {
 func getDefaultPolicyPath() string {
 	return trust.DefaultPolicyPath(&types.SystemContext{})
 }
+
+var typeDescription = map[string]string{"insecureAcceptAnything": "accept", "signedBy": "signed", "reject": "reject"}
+
+func trustTypeDescription(trustType string) string {
+	trustDescription, exist := typeDescription[trustType]
+	if !exist {
+		logrus.Warnf("invalid trust type %s", trustType)
+	}
+	return trustDescription
+}
+
+func getPolicyOutput(policyContentStruct trust.PolicyContent, systemRegistriesDirPath string) (map[string]map[string]interface{}, map[string]trust.ShowOutput, error) {
```

Comment:
Non-blocking: This computes both the text and JSON output at the same time, when only one of the return values will be used.

It would be a bit cleaner to have two separate formatters that each computes only one of the outputs, and then call only the right one.

(And to do that, move even more of the shared code, like the call to `LoadAndMergeConfig`/`HaveMatchRegistry` and the `GPGId` computation, from this CLI package into the backend `trust` subpackage, and have it returned by `trust.GetPolicy`.